src/ui/ui_apps/ui_app.js

```diff
@@ -22,6 +21,10 @@ export class UiApp {
       throw new Error('Every app must specify an id');
     }
 
+    if (spec.injectVars) {
```

Comment:
__issue:__ it seems this `if` will never trigger since `injectVars` is always "trimmed" in `ui_export_types/ui_apps.js` now. If I'm not mistaken, we should do this check there instead?